### PR TITLE
Add GoLand IDE files to .gitignore

### PR DESCRIPTION
**Pull request by roytman**
Signed-off-by: Alexey Roytman <roytman@il.ibm.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -79,3 +79,5 @@ tags
 .vscode/*
 .history
 # End of https://www.gitignore.io/api/go,vim,emacs,visualstudiocode
+#IDE (GoLand) specific
+.idea/
```
Comment (pliurh) [inline .gitignore:83]: nit, please add a new line at the end of the file.

Comment (roytman) [reply .gitignore]: Done

**Pull request merged**